ote_cli/ote_cli/tools/eval.py

```diff
@@ -20,6 +20,7 @@
 import json
 
 from ote_sdk.configuration.helper import create
+from ote_sdk.entities.datasets import DatasetEntity
```

Comment:
Do we still need this?

Reply:
Updated both MPA commit change and removing the ote side code change